chattore/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    val kotlinVersion = "2.1.10"
```

Comment:
The subprojects don't need the versions specified, it should be enough to do `kotlin("jvm")` and `id("com.gradleup.shadow")` etc

Reply:
`kotlin("kapt")` and `kotlin("plugin.serialization")` may be cleaner here too